packages/extension/src/newtab/MainFeedPage.tsx

```diff
@@ -119,7 +123,13 @@ export default function MainFeedPage({
             />
           }
           navChildren={!isSearchOn && <ShortcutLinks />}
-          besideSearch={<ShortcutLinks className="ml-auto" />}
+          shortcuts={
+            <ShortcutLinks
+              className={
+                layout === FeedLayoutEnum.Control ? 'ml-auto' : 'mt-4 w-fit'
```

Comment:
wouldn't this depend on the feedName as well?

Reply:
These shortcuts are naturally not available on search feed and are only seen on the extension which is only on all the valid feeds.